app-misc/asciinema/asciinema-2.0.2.recipe

```diff
@@ -46,14 +45,11 @@ BUILD()
 
 INSTALL()
 {
-	sed -i \
-		-e "s|documentation/man|$relativeManDir|" \
-		-e "s|documentation/packages/asciinema|$relativeDocDir|" \
-		setup.py
-	python3 setup.py install --prefix=$prefix --root=/ --skip-build
-}
+	sed -i "s|share/doc/asciinema|$relativeDocDir|
+		s|share/man|$relativeManDir|" setup.py
+	python3 setup.py install --prefix="$prefix" --root=/ --skip-build
 
-TEST()
-{
-	python3 setup.py test
+	packageEntries $pythonPackage \
```

Comment:
But there aren't any `$pythonPackage` in sight? I don't think this is needed.

Reply:
Ah, I forgot why I didn't include it in the first place and assumed that I forgot to. I'll fix it as soon as I get back to my computer.